src/evm/process-sponsor-wallet.test.ts

```diff
@@ -245,7 +261,10 @@ describe('processSponsorWallet', () => {
     const fulfillPspBeaconUpdateOnceSpy = jest
       .fn()
       .mockImplementationOnce(fulfillPspBeaconUpdateMock)
-      .mockImplementationOnce((_nameOrSignatureOrSighash: string) => {
+      .mockImplementationOnce(() => {
```

Comment:
After the upgrade to new promise utils, this test got broken and I've figured that the function  wasn't retried properly before.